VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/judge-waiting-room/judge-waiting-room.component.html

```diff
@@ -17,6 +17,10 @@
   </div>
 </div>
 
+<ng-container *ngIf="connected && showVideo">
```

Comment:
Please move this inside the video-container as discussed previously

Reply:
np, will move